src/imitation/algorithms/preference_comparisons.py

```diff
@@ -463,12 +770,20 @@ def __init__(
                 condition, and can seriously confound evaluation. Read
                 https://imitation.readthedocs.io/en/latest/guide/variable_horizon.html
                 before overriding this.
+            seed: seed to use for initializing subcomponents such as fragmenter.
+                Only used when default components are used; if you instantiate your
+                own fragmenter, preference gatherer, etc., you are responsible for
+                seeding them!
```

Comment:
Mention return value in docstring?

Reply:
I assume you mean for the `train()` method? Added it there

Reply:
yes, sorry for commenting in the wrong place, thanks.